spring-cloud-contract-verifier/src/test/java/org/springframework/cloud/contract/verifier/builder/XmlBodyVerificationBuilderTest.java

```diff
@@ -0,0 +1,33 @@
+package org.springframework.cloud.contract.verifier.builder;
+
+import org.junit.Test;
+import org.springframework.cloud.contract.spec.Contract;
+import org.springframework.cloud.contract.spec.internal.BodyMatchers;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertTrue;
+
+public class XmlBodyVerificationBuilderTest {
+
+	private static final String xml = "<customer>\n" +
+			"      <email>customer@test.com</email>\n" +
+			"    </customer>";
+
+	@Test
+	public void shouldAddXmlProcessingLines() {
+		//Given
+		XmlBodyVerificationBuilder builder = new XmlBodyVerificationBuilder(new Contract(), Optional.of(";"));
+		BlockBuilder blockBuilder = new BlockBuilder(" ");
+		BodyMatchers matchers = new BodyMatchers();
+		//When
+		builder.addXmlResponseBodyCheck(blockBuilder, xml, matchers, xml, true);
+		//Then
+		String test = blockBuilder.toString();
+		assertTrue(test.contains("DocumentBuilderFactory builderFactory = DocumentBuilderFactory.newInstance();"));

```

Comment:
Can we use AssertJ instead of JUnit assertions?

Reply:
Done

Reply:
done